data/idp_certificates/eduid.crt

```diff
@@ -0,0 +1,17 @@
+MIIDDDCCAfSgAwIBAgIVALoCDnpVfF76D6WKc9mDL9jtreW1MA0GCSqGSIb3DQEB 
```

Comment:
Is it a real certificate? Can it be public?

Reply:
Yes it's the public certificate of identity providers, available here:

http://metadata.aai.switch.ch/metadata.switchaai+idp.xml